tests/codemods/test_break_or_continue_out_of_loop.py

```diff
@@ -0,0 +1,58 @@
+from codemodder.codemods.test import BaseCodemodTest
+from core_codemods.break_or_continue_out_of_loop import BreakOrContinueOutOfLoop
+
+
+class TestBreakOrContinueOutOfLoop(BaseCodemodTest):
+    codemod = BreakOrContinueOutOfLoop
+
+    def test_name(self):
+        assert self.codemod.name == "break-or-continue-out-of-loop"
+
+    def test_continue(self, tmpdir):
```

Comment:
an important test case is within the issue writeup, could you add it?

```
def print_even_numbers():
    for i in range(100):
        if i % 2 == 0:
            print(i)
    else:
        continue  # [not-in-loop]
```
removing the entire else clause would be the best thing here, but doing nothing is also an option.

Reply:
This case is covered in other tests, but I will add it nonetheless. It will replace the `continue` with `pass`.

Reply:
I was mistaken, I didn't realize loops could have an `else` clause. I went ahead and also added logic to remove the else entirely if empty.